builder/ConfTargets.cmake

```diff
@@ -109,6 +109,7 @@ cmake_dependent_option(
 option( MFX_ENABLE_VP9_VIDEO_ENCODE "Enable VP9 encoder?" ON)
 
 option( MFX_ENABLE_MCTF "Build with MCTF support?"  ${MFX_1_25_OPTIONS_ALLOWED} )
```

Comment:
I commented before: "MCTF actually depends on ASC and can't work without it." This means that you should rewrite as:
```
cmake_dependent_option(
  MFX_ENABLE_MCTF "Enable MCTF?" ${MFX_1_25_OPTIONS_ALLOWED}
  "MFX_ENABLE_ASC;MFX_ENABLE_KERNELS" OFF)
```
And eventually have option for ASC listed first.